ocs_ci/framework/conf/default_config.yaml

```diff
@@ -91,6 +91,7 @@ REPORTING:
   ocs_must_gather_image: "quay.io/rhceph-dev/ocs-must-gather"
   default_ocs_must_gather_latest_tag: 'latest-4.5'
   gather_on_deploy_failure: true
+  gather_on_deploy_success: true
```

Comment:
Why should be must gather collected for successful deployment by default?

Reply:
When jobs and tests complete without failures, no logs are collected. And sometimes there are still some details about the cluster and the execution are needed.
So collecting must-gather after a successful deployment could be a good way to be aware of the cluster's situation immediately after deployment and we would have deployment logs.

It was requested and discussed [here](https://github.com/red-hat-storage/ocs-ci/issues/2598)

Reply:
Ok :+1: